x-pack/plugins/upgrade_assistant/public/components/tabs/checkup/deprecations/reindex/flyout/progress.test.tsx

```diff
@@ -33,19 +33,35 @@ describe('ReindexProgress', () => {
     Array [
       Object {
         "status": "incomplete",
-        "title": "Setting old index to read-only",
+        "title": <FormattedMessage
+          defaultMessage="Setting old index to read-only"
+          id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.reindexingChecklist.readonlyStepTitle"
+          values={Object {}}
```

Comment:
Are the values required here?

Reply:
There are others that also don't have templates which define a default value.

Reply:
This is the test snapshot, so this empty object is just the defaultProp.